gcloud-java-core/src/test/java/com/google/gcloud/SerializationTest.java

```diff
@@ -52,6 +53,8 @@ public Builder toBuilder() {
   private static final Identity IDENTITY = Identity.allAuthenticatedUsers();
   private static final PageImpl<String> PAGE =
       new PageImpl<>(null, "cursor", ImmutableList.of("string1", "string2"));
+  private static final SigningException SIGNING_EXCEPTION =
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/mziccard/gcloud-java/file/2362596895/issues/source?bid=3123103&fileBranchId=3219607#l56)